### ISSUE TITLE
How to test an app from a specific branch?

### ISSUE THREAD

**Issue by yzernik** [getumbrel/umbrel-apps#40]
The instructions here don't say how to install an app for testing from a git branch, only from a specific fork:
https://github.com/getumbrel/umbrel-apps#32-testing-on-umbrel-os-raspberry-pi-4

**Comment by lukechilds** [comment]
We're working on adding some commands to make this a little simpler. For now you should be able to cd into the repo directory in `repos/` and manually add new remote or checkout branches. You may need to prepend `sudo` to the git commands if you get permission issues. You should run `sudo chown -R 1000:1000 repos/` afterwards to reset the permissions.

Sorry for the hassle, we'll make this simpler soon!

**Comment by nevets963** [comment]
Hey @yzernik 

We just released Umbel v0.5.1 that makes this now easy.

Here are some example commands you can now run:
```
~/umbrel/scripts/repo checkout http://github.com/coolapp/umbrel-apps
~/umbrel/scripts/repo checkout coolapp/umbrel-apps                 (will use github.com via https)
~/umbrel/scripts/repo checkout coolapp/umbrel-apps#staging         (#<branch-name>)
```

**Comment by yzernik** [comment]
Thanks @nevets963. I'll close this now.

Do you think this should be in the README?
